pages/postop.tsx

```diff
@@ -33,48 +33,78 @@ function PostOp() {
       case 'repeatKetones':
         return (
           <QuestionCard
-            question="Repeat ketones and BGL in PACU"
-            primaryAction={{ copy: 'Ketones > 1.0 mmol/L', action: sendYes }}
-            secondaryAction={{ copy: 'Ketones <= 1.0 mmol/L', action: sendNo }}
-          />
+            question="Repeat Ketones and Glucose in PACU"
+            primaryAction={{ copy: 'Ketones > 1.0', action: sendYes }}  //I think this might be round the wrong way
```

Comment:
Thanks, let's fix this in a separate PR

```suggestion
            primaryAction={{ copy: 'Ketones > 1.0', action: sendYes }}
```